src/uniformscaling.jl

```diff
@@ -217,6 +217,22 @@ function (-)(J::UniformScaling{<:Complex}, A::Hermitian)
     return B
 end
 
+for f in (:+, :-)
+    @eval begin
+        function $f(A::AdjOrTransAbsMat, J::UniformScaling)
+            checksquare(A)
+            op = wrapperop(A)
+            op($f(op(A), op(J)))
```

Comment:
```suggestion
            op($f(parent(A), op(J)))
```
?

Reply:
In most cases, these are equivalent. The difference might arise in nested adjoints, where the wrapper function might unwrap multiple layers, whereas `parent` only unwraps one layer. Such cases are uncommon, though.